casepro/msgs/models.py

```diff
@@ -417,6 +429,15 @@ def get_history(self):
         """
         return self.actions.select_related('created_by', 'label').order_by('-pk')
 
+    def get_lock(self, user_id):
```

Comment:
Sorry for not noticing this in last review but why is this an id rather than a user object?